test/runtime/adapter-config.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var juttle_test_utils = require('./specs/juttle-test-utils');
+var check_juttle = juttle_test_utils.check_juttle;
+var path = require('path');
+var expect = require('chai').expect;
+var _ = require('underscore');
+var adapters = require('../../lib/runtime/adapters');
+
+
+function configure(name, conf) {
+    conf = _.clone(conf);
+    conf.path = path.resolve(__dirname, '../runtime/test-adapter-conf');
```

Comment:
For completeness, I think it would be good to test that the adapter loads as expected when the path isn't supplied, and I find it kinda hackish that we test the case of assigning a `path` property to an array since that just seems dirty.

How about if we change the adapters implementation somewhat so that the internal `adapterModulePath` could be overridden by the unit test.

Then you could test the various configuration options here both with the implicit path (derived from your overridden module path handler) and with the explicit path in the cases where the configuration is an object.